### DISCUSSION TITLE
Maybe use the Accent color from OS system preferences

### DISCUSSION THREAD

**Discussion by vladstudio** [notaapp/talk#268] [Feature Requests]
Nota seems to only use one saturated color - light-blue for selected folder/file background.

If I change my accent color in OS system preferences (General tab), Nota is not affected. No big deal but a bit un-ideal:

<img width="529" alt="image" src="https://user-images.githubusercontent.com/914320/180386532-d3fa3b7b-619d-430b-9d0d-d5a1bd2442df.png">

Suggestion: take hue color component from OS accent.

This idea may conflict with any theming ideas you might have, though.

**Comment by astoilkov** [comment]
I like the idea. Do you know some nonnative apps that use that color? What do they do in dark mode? Thanks!

**Reply by vladstudio** [reply]
I am not sure about non-native, but here is how Forklift app reacts to OS accent color change (note the background of Path bar):

<img width="471" alt="image" src="https://user-images.githubusercontent.com/914320/180402686-65b9adf0-d435-4e7e-9be9-0e6682019d5f.png">

**Reply by astoilkov** [reply]
Sorry, by nonnative I meant apps not built by Apple. Thanks. We can experiment if all colors look good enough in both themes.

It will be a very cool improvement. I think my co-founder @erusev also wanted this a few months ago.
